javascript-functions/main.js

```diff
@@ -0,0 +1,45 @@
+function convertMinutesToSeconds(minutes) {
+  const minutesToSeconds = minutes * 60;
+  return minutesToSeconds;
+}
+const convertMinutesToSecondsResult = convertMinutesToSeconds(300);
+console.log('convertMinutesToSeconds Result:', convertMinutesToSecondsResult);
+
+function greet(name) {
+  const greetings = 'Hey, ' + name;
+  return greetings;
+}
+const greetResult = greet('Beavis');
+console.log('greet Result:', greetResult);
+
+function getArea(width, height) {
+  const area = width * height;
+  return area;
+}
+const getAreaResult = getArea(17, 42);
+console.log('getArea Result:', getAreaResult);
+
+function getFirstName(person) {
+  person = {
+    firstName: 'Lelouch',
+    lastName: 'Lamperouge',
+  };
```

Comment:
You don't want to define `person` inside of this function. `person` should be getting passed in when this function is called

Reply:
Okay, I'll correct it thanks for the help